apps/go-mod-validator/src/index.ts

```diff
@@ -0,0 +1,96 @@
+import { validateDependency } from "./github";
+import { getDependenciesMap } from "./deps";
+import * as core from "@actions/core";
+import minimist from "minimist";
+import { Octokit } from "octokit";
+
+const smartContractKitPrefix = "github.com/smartcontractkit";
+
+function getContext() {
+  const argv = minimist(process.argv.slice(2));
+  const { local, tokenEnv, goModDir } = argv;
+
+  const dir = local ? goModDir || `"$(pwd)"` : core.getInput("go-mod-dir");
+
+  const githubToken = local
+    ? process.env[tokenEnv] || ""
+    : core.getInput("github-token");
```

Comment:
`getInput` actually parses env vars according to the specified format here: https://github.com/actions/toolkit/blob/main/packages/core/src/core.ts#L128
We can set env vars locally according to the same spec and remove the need for arg parsing that way